integration/kubernetes/helm-chart/alluxio/templates/csi/driver.yaml

```diff
@@ -0,0 +1,20 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+{{ if .Values.csi.enabled -}}
+apiVersion: storage.k8s.io/v1beta1
+kind: CSIDriver
+metadata:
+  name: alluxio
+spec:
+  attachRequired: false
+  podInfoOnMount: true
```

Comment:
Not sure if these should be parameterized?

Reply:
These parameters used by developer. Admin/users don't need to take care